the-collab-lab/tcl-10-smart-shopping-list
# Main styling

## Description
- updated input fields to material-ui components
- added top navbar
- added a copy modal which copies the share code to the clipboard 
- added logout functionality to the navbar which returns the user to the homepage
- adjusted clear icon button
- remove unused styles leftover from `create-react-app`


We wanted the styling of this app to be straightforward and modern.  We wanted to add more functionality to enable users to access multiple lists by logging out and to make it easier for users to share list tokens.

## Related Issue



## Acceptance Criteria



## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |
|  ✓ | :sparkles: Update styling     |

## Updates

### Before
![image](https://user-images.githubusercontent.com/50968795/94217102-2c0cf500-feaf-11ea-8a3e-a59fa7dacb7d.png)
![image](https://user-images.githubusercontent.com/50968795/94217113-33340300-feaf-11ea-90fe-afdffe5b91ce.png)

### After
![image](https://user-images.githubusercontent.com/50968795/94217124-38914d80-feaf-11ea-881b-daa44baf6f07.png)
![image](https://user-images.githubusercontent.com/50968795/94217135-3dee9800-feaf-11ea-95c4-045184d19ff5.png)
![image](https://user-images.githubusercontent.com/50968795/94217141-42b34c00-feaf-11ea-8994-8f0be13d44e9.png)


## Testing Steps / QA Criteria

- Create a new list or use an existing token
- Make sure that the clear button still works when searching for an item.
- Check that you can copy the share token by clicking on the share icon in the navbar and clicking the `copy` button.
- Check that you can log out by clicking the logout icon in the navbar and that you are automatically directed back to the homepage.
- Check that the share and logout icons only appear once you've entered a share token and disappear when you exit a list.
